about.html

```diff
@@ -0,0 +1,178 @@
+              >
+                <path
+                  d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"
+                />
+              </svg>
+            </a>
+  
+            <span class="span">Logout</span>
+            <span class="span">MyPage</span>
+            <span class="span">English</span>
+          </div>
+        </header>
+    </nav>
+
+    <div class="container-fluid"></div>
+        <div class="navdiv sticky-top"></div>
+          <nav class="navbar d-none d-md-flex maxwidth ">
+             <a href="index.html"><img class="navbar-brand" src="./Images/Spectra-logo.webp" alt="Spectra-logo logo"> </a> 
+             <div class="navlinks d-md-flex justify-content-between align-items-center">          
+               <a href="index.html">Home</a>
+               <a href="#" class="active">About</a>
+               <a href="Services">Services</a>
+               <a href="#" class="nav-link disabled">Contact</a> 
+               <button type="button">Book Now</button>  
+             </div>
+           </nav>
+        
+           <div class="menu">
+
+			<div id="myNav" class="overlay">
+
+				<a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+
+				<div class="overlay-content">
+					<a href="index.html" onclick="closeNav()">Home</a>
+					<a href="#" onclick="closeNav()">About</a>
+					<a href="#" onclick="closeNav()">Services</a>
+                    <a href="#" onclick="closeNav()">Contact</a>
+				</div>
+			</div>
+           <nav class="d-flex justify-content-between d-md-none maxwidth smallnav pl-3 navbar ">        
+            <a href="index.html"><img class="navbar-brand-img1" src="./Images/Spectra-logo.webp" alt="Spectra logo"></a>         
+            <span  onclick="openNav()" class="fas fa-bars" data-toggle="collapse" data-target="#collapsibleNavbar"></span>
+            <div class="collapse navbar-collapse py-2" id="collapsibleNavbar">
+                <ul class="navbar-nav pl-3 float-right bg-light">
+                    <li class="navlinks "><a href="index.html">Home</a></li>
+                    <li class="navlinks"><a href="about.html">About</a></li>
+                    <li class="navlinks"><a  href="Services">Services</a></li>
+                </ul>
+            </div>          
+        </nav>
+     </div>  
+
+     <section class="about">
+        <div class="about-content">
+            <h1>Welcome to Spectra Beauté</h1>
+          <h2 class="message">"We believe in our responsibility to inspire"</h2>
+        </div>
```

Comment:
- Kindly fix the overflow on the page for a better User Interface
![screenshot (1)](https://user-images.githubusercontent.com/44624138/181954041-dde06aea-9846-4d53-9951-06e659b30f22.png)